mypy/stubtest.py

```diff
@@ -630,7 +630,11 @@ def verify_typeinfo(
         return
 
     yield from _verify_final(stub, runtime, object_path)
-    yield from _verify_disjoint_base(stub, runtime, object_path)
+
+    # TODO: Once PEP 800 gets accepted, remove the conditional (always verify)
+    if "--ignore-disjoint-bases" not in sys.argv[1:]:
```

Comment:
Could you implement this more like the `args.ignore_positional_only` check? (Also pretty hacky but at least consistent.)